_pages/software-tools/google-drive.md

```diff
@@ -37,11 +37,11 @@ Make sure you set the permissions on new documents so that the appropriate peopl
 
 * If another agency employs Google Docs, documents can be shared directly with them.
 
-* If the agency does not use Google Docs, [here are approved instructions for sharing documents with them](https://insite.gsa.gov/portal/content/517805). Agency staff can also create a google account using their work email [by using this form](https://accounts.google.com/SignUpWithoutGmail?hl=en) or creating a [GSA Affiliated Customer Account (GACA)](https://insite.gsa.gov/portal/content/517805?term=gaca). Ask in [#teamops](https://gsa-tts.slack.com/messages/teamops) or [#it-issues](https://gsa-tts.slack.com/messages/it-issues) if you need help.
+* If the agency does not currently use Google Drive, agency staff may be able to create a google account using their work email [by using this form](https://accounts.google.com/SignUpWithoutGmail?hl=en). If Staff aren't able to access that form, that likely means their agency expressly blocks Google Drive. This is especially relevant if you are creating a form that is intended for staff of another agenc (see next bullet).
 
-### Google Forms
+* If you create a Google form that is intended for employees of another agency, keep in mind that in a few agencies, Google Forms are blocked. Among the largest agencies that are known to currently block Forms are: DHS, VA, and EOP. In the case that you intend to involve those agencies in an initiative where a Google Form is being used, it's smart to create a version in another format, like a Word doc or fillable PDF.
```

Comment:
We should spell out those agency names for clarity. I'm not sure what EOP is

Reply:
done